### PR TITLE
Login form update

### PR DESCRIPTION
**Pull request by KenStephens**
#Snapshot
[Screenshot](Screenshot 2024-05-14 120259)

#Changes
- Changed form data
- Added a banner image to combat dead space on login page
- UseNavigate hook to transfer user-data on successful login to home page

#How to test
- navigate to "/login"

### CODE REVIEW

File: src/components/Login.jsx [hunk 1]
```diff
@@ -1,5 +1,98 @@
...
+        <div className="row justify-content-center">
+          <div className="col-md-3">
+            <Formik
+              enableReinitialize={true}
+              initialValues={loginData}
+              onSubmit={onSubmit}
+              validationSchema={loginFormSchema}
+            >
+              <Form>
+                <div className="">
```
Comment (woodmx) [inline src/components/Login.jsx]: if this div is not using this classname, can you remove it

File: src/components/Login.jsx [hunk 2]
```diff
@@ -1,5 +1,98 @@
...
+                    id="username"
+                    placeholder="Username"
+                    className="form-control"
+                  />
+                  <ErrorMessage
+                    name="username"
+                    components="div"
+                    className="has error"
+                  />
+                  <div></div>
```
Comment (woodmx) [inline src/components/Login.jsx]: whats the reason for this empty div

File: src/components/Login.jsx [hunk 3]
```diff
@@ -1,5 +1,98 @@
...
+                    id="password"
+                    placeholder="Password"
+                    className="form-control "
+                  />
+                  <ErrorMessage
+                    name="Password"
+                    components="div"
+                    className="has-error"
+                  />
+                  <div></div>
```
Comment (woodmx) [inline src/components/Login.jsx]: same here

**Review by woodmx — CHANGES_REQUESTED** [review]
changes requested

File: src/components/NavBar.module.css [hunk 4]
```diff
@@ -1,29 +1,33 @@
...
+  font-family: "Merriweather", serif;
 }
 
-.navLink:hover {  
+.navLink:hover {
   color: var(--secondary-color) !important;
+  font-family: "Merriweather", serif;
   font-size: 1.05rem;
   transition: 0.2s;
 }
```
Comment (jjcanell77) [inline src/components/NavBar.module.css]: You don't need to add the font-family to the hover as you added it to the main class.

File: src/components/NavBar.module.css [hunk 5]
```diff
@@ -1,29 +1,33 @@
...
-.endText:hover  {
-  color: var(--support-color)!important;
+.endText:hover {
+  font-family: "Merriweather", serif;
+  color: var(--support-color) !important;
   font-size: 1.05rem;
   transition: 0.2s;
-}
\ No newline at end of file
+}
```
Comment (jjcanell77) [inline src/components/NavBar.module.css]: Same here. You don't need the extra font-family change.

File: src/components/footer/Footer.module.css [hunk 6]
```diff
@@ -1,80 +1,80 @@
 :root {
-     --main-color: #062d1d;
-     --secondary-color: #E3AA4D;
-     --support-color: #ffffff;
+  --main-color: #062d1d;
+  --secondary-color: #e3aa4d;
```
Comment (jjcanell77) [inline src/components/footer/Footer.module.css:3]: Its the same color, why did you need to change this?

**Review by jjcanell77 — CHANGES_REQUESTED** [review]
Overall looks good.

File: src/components/Login.jsx (hunk 2 above)
Comment (KenStephens) [reply src/components/Login.jsx]: so the error message has its own line when a username is not input

File: src/components/NavBar.module.css (hunk 5 above)
Comment (KenStephens) [reply src/components/NavBar.module.css]: done

File: src/components/footer/Footer.module.css (hunk 6 above)
Comment (KenStephens) [reply src/components/footer/Footer.module.css:3]: done

File: src/components/NavBar.module.css (hunk 4 above)
Comment (KenStephens) [reply src/components/NavBar.module.css]: done

**Review by jjcanell77 — APPROVED** [review]

**Review by woodmx — APPROVED** [review]

**Comment by jmedina1329** [comment]
Please update this branch with main

**Comment by BWFischer** [comment]
Please update this branch as it has conflicts that must be resolved.
